lesson_19/homework_19_2.py

```diff
@@ -0,0 +1,46 @@
+import os.path
+import sys
+import requests
+from asgiref.timeout import timeout
+from requests import HTTPError
+from flask import request
+import json
+
+"""Враховуючи документацiю яку наведено нижче вам потрiбно написати код який використовуючи модуль
+request зробить через POST upload якогось зображення на сервер, за допомогою GET отримає посилання
+на цей файл и потiм за допомогою DELETE зробить видалення файлу з сервера"""
+
+url = "http://127.0.0.1:8080"
+image = "/Users/natalynazarenko/PycharmProjects/pythonProject1/lesson_19/uploads/44.png"
+
+
+def post_image(url, image, timeout):
+     if not os.path.exists(image):
+         print("Not found")
+
+     try:
```

Comment:
try-except block with raise for status could be hidden into some helper function aka do_request. This functions accepts name of operation ( get/post/delete) and parameters ( args/kwargs) and returns response  for example